data_classes_test.py

```diff
@@ -0,0 +1,52 @@
+"""
+Unit tests for data classes
+
+Implemented:
+    Data
+
+TODO:
+    DataController
+    Dataset
+    Portfolio
+"""
+
+import unittest
+from datetime import datetime
+from thousandaire.dataset import Data
+
+class TestData(unittest.TestCase):
+    """
+    Unit test object for Data.
+    """
+    def setUp(self):
+        self.data = Data('test', ['buy', 'sell'])
+
+    def test_fields(self):
+        """
+        Test fields.
+        """
+        self.assertEqual(self.data.fields, ['date', 'buy', 'sell'])
+
+    def test_append(self):
```

Comment:
Is it possible to test invalid cases here as well?
For example, the pushed item is not a tuple, or is a tuple with wrong dimension, etc.